diff -Nru bison-1.28/doc/bison.texinfo bison-1.28.new/doc/bison.texinfo
--- bison-1.28/doc/bison.texinfo	Thu Jan 14 16:08:17 1999
+++ bison-1.28.new/doc/bison.texinfo	Wed Jul  7 15:11:43 1999
@@ -37,9 +37,10 @@
 
 @ifinfo
 @format
-START-INFO-DIR-ENTRY
-* bison: (bison).	GNU Project parser generator (yacc replacement).
-END-INFO-DIR-ENTRY
+@dircategory Programming tools:
+@direntry
+* bison: (bison).			The GNU parser generator
+@end direntry
 @end format
 @end ifinfo
 
