--- bison-1.875/data/yacc.c.orig	Sat Dec 28 09:36:02 2002
+++ bison-1.875/data/yacc.c	Tue Jan 28 20:57:50 2003
@@ -1114,7 +1114,11 @@
   /* Suppress GCC warning that yyerrlab1 is unused when no action
      invokes YYERROR.  */
 #if defined (__GNUC_MINOR__) && 2093 <= (__GNUC__ * 1000 + __GNUC_MINOR__)
+  /* but it's harmful in C++ code 
+   * (gcc 3.2.1 complained about erroneous yylerrsp declaration) */
+#  ifndef __cplusplus
   __attribute__ ((__unused__))
+#  endif
 #endif
 
 ]b4_location_if([  yylerrsp = yylsp;
